tutor/jobs.py

```diff
@@ -126,11 +126,18 @@ def set_theme(
         return
     python_code = "from django.contrib.sites.models import Site"
     for domain_name in domain_names:
+        if len(domain_name) > 50:
+            fmt.echo_alert(
+                f"The domain {domain_name} is too long to be used for site name, "
+                "truncating to 50 characters. Please update the site name using admin."
+            )
```

Comment:
I think it's great that you added a warning, but we should modify the wording. It's not really a big deal if the site displayed name is incorrect, so we should not suggest to modify it in the admin. Instead, I propose the following:

> Assigning a theme to a site with a long (> 50 characters) domain name. The displayed site name will be truncated to 50 characters.

Reply:
Thanks @regisb . Updated the commit with your suggestion.